PHPCI/Migrations/20150131075425_archive_project.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class ArchiveProject extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $project = $this->table('project');
+        $project->addColumn('archived', 'boolean');
```

Comment:
Everywhere else, the column is referred to as `archive` - here you've used `archived`.

As it happens, I actually prefer "archived".